std/jwt/README.md

```diff
@@ -0,0 +1,102 @@
+# jwt
+
+Create and verify JSON Web Tokens.
+
+## JSON Web Token
+
+### create
+
+Takes a `payload`, `key` and `header` and returns the url-safe encoded `jwt`.
+
+```typescript
+import { create } from "https://deno.land/std/jwt/mod.ts";
+
+const payload = { foo: "bar" };
+const key = "secret";
+
+const jwt = await create(payload, key); // eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJmb28iOiJiYXIifQ.4i-Q1Y0oDZunLgaorkqbYNcNfn5CgdF49UvJ7dUQ4GVTQvpsMLHABkZBWp9sghy3qVOsec6hOcu4RnbFkS30zQ
```

Comment:
is there any reason for using the async function?

This module does not seem to use the wasm module, there is no need to use async function.

Reply:
> Although not necessary now, verify and create return promises because we
expect coming encryption functions to return promises in the future. E.g. for
RSA algorithms.

good to see.